logger_test.go

```diff
@@ -0,0 +1,36 @@
+package analytics
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"testing"
+)
+
+// This test ensures that the interface doesn't get changed and stays compatible
```

Comment:
Yup! Not a big deal though, since either way it'll break when it's not working and and not break when it's working so fine to leave it as is.